fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -867,8 +867,13 @@ setDefaultStorageVolumeStatement
 
 updateFailPointStatusStatement
     : ADMIN DISABLE FAILPOINT string (ON BACKEND string)?
-    | ADMIN ENABLE FAILPOINT string (WITH INTEGER_VALUE TIMES)? (ON BACKEND string)?
-    | ADMIN ENABLE FAILPOINT string (WITH DECIMAL_VALUE PROBABILITY)? (ON BACKEND string)?
+    | ADMIN ENABLE FAILPOINT string (ON BACKEND string)?
+    | ADMIN ENABLE FAILPOINT string WITH INTEGER_VALUE TIMES (ON BACKEND string)?
+    | ADMIN ENABLE FAILPOINT string WITH DECIMAL_VALUE PROBABILITY (ON BACKEND string)?
+    | ADMIN DISABLE FAILPOINT string ON FRONTEND
+    | ADMIN ENABLE FAILPOINT string ON FRONTEND
+    | ADMIN ENABLE FAILPOINT string WITH INTEGER_VALUE TIMES ON FRONTEND
+    | ADMIN ENABLE FAILPOINT string WITH DECIMAL_VALUE PROBABILITY ON FRONTEND
```

Comment:
```suggestion
    updateFailPointStatusStatement
    : ADMIN (DISABLE | ENABLE) FAILPOINT string 
      (WITH (times=INTEGER_VALUE TIMES | prob=DECIMAL_VALUE PROBABILITY))? 
      (ON (BACKEND string | FRONTEND))?
    ;

```

Reply:
This is very concise, but a little hard to read?